src/pages/en/guides/content-collections.mdx

```diff
@@ -121,6 +121,19 @@ If you **do not** already extend Astro's `strict` or `strictest` recommended Typ
 }
 ```
 
+:::note[Not using TypeScript?]
+If you use `.js` or `.mjs` files in an Astro project, you can enable IntelliSense and type checking in your editor by enabling `allowJs` in your `tsconfig.json`:
+```json title="tsconfig.json" ins={5}
```

Comment:
````suggestion
If you use `.js` or `.mjs` files in an Astro project, you can enable IntelliSense and type checking in your editor by enabling `allowJs` in your `tsconfig.json` in addition to adding `strictNullChecks` as described above:
```json title="tsconfig.json" ins={4-5}
````
Is this overkill?

Reply:
Sorry, I just saw this suggestion. Do you think this still fits in new the commit I made?

Reply:
Ooh, I like that! I think this looks good!

Reply:
Awesome! Just waiting for the ✅s then.